### ISSUE TITLE
why the package name is heaven

### ISSUE THREAD

**Issue by cjz9032** [pixijs/pixi-swf#2]
https://github.com/pixijs/pixi-swf/blob/e98e5b8d39533b11e1b51895d07e98bbefdd29f0/package.json

"name": "pixi-heaven"

**Comment by ivanpopelyshev** [comment]
yeah, should be pixi-swf, but I still didnt have time to update it to latest version that i use in https://www.facebook.com/thehousehold :)
